README.md

```diff
@@ -16,11 +16,11 @@ commented out
 -->
 
 ## Requirements
-*   Node 4
+*   Node version 14.15.5 or higher
```

Comment:
From a quick glance, we're going to need to re-evaluate the decision to bump the required version of node here. With any public facing library, it is important to keep as much backwards compatibility as possible, that is why you saw the legacy syntax on those unit tests. We are in the process of evaluating dropping support for versions 4-7, however this isn't something to be done lightly as it will have far reaching impacts on our users. That being said, making the minimum required version be 14.15.5 will be too strict for us at this time.